app/Models/Entity.php

```diff
@@ -449,7 +449,7 @@ public function hasPins(): bool
      */
     public function postPositionOptions($position = null): array
     {
-        $options = [null => __('characters.placeholders.dont_change'), 1 => __('maps/layers.placeholders.position')];
+        $options = [null => __('entities.dont_change'), 1 => __('maps/layers.placeholders.position')];
```

Comment:
Please place it in `posts.php` where we already collect everything related to posts, and not in `entities.php` which is only related to the various entity types :)